src/software/backend/robot_status.h

```diff
@@ -84,3 +86,12 @@ struct RobotStatus
     // The FPGA bitstream build ID.
     uint32_t fpga_build_id;
 };
+
+/**
+ * Converts RobotStatus to TbotsRobotMsg
+ * Does not convert ChipperKickerStatus,
+ * DriveUnits, and NetworkStatus due to insufficient info
+ *
+ * @param robot_status The RobotStatus
+ */
+TbotsRobotMsg convertRobotStatusToTbotsRobotMsg(RobotStatus* robot_status);
```

Comment:
Why are we passing `RobotStatus` here by pointer? Passing by const reference is fairly common in our codebase if the concern is avoiding a copy here, or just a reference if we need to modify it in-place.

Reply:
Changed to pass by const reference. Just curious, what's the reason for const reference?